src/test/java/com/pro_crafting/tools/jasperreport/JasperReportCompilerTest.java

```diff
@@ -334,7 +333,7 @@ private JasperMojoConfiguration buildDefaultConfiguration() {
 
         configuration.xmlValidation = true;
```

Comment:
Since you deleted the configuration handling in JasperReportCompiler#134 I guess we should also delete the possibility to configure it. I don't see why one would want to validate the xml since jackson will error out on any error anyway.